ArcGISRuntimeSDKQt_CppSamples/Maps/ReadGeopackage/ReadGeopackage.json

```diff
@@ -0,0 +1,8 @@
+{
+    "source" :  "ReadGeoPackage.qml",
+    "otherSourceFiles" :  [
+        "ReadGeoPackage.h",
+        "ReadGeoPackage.cpp"
+    ],
+    "name" : "Read geoPackage"
```

Comment:
I think the C++ sample needs the `dataItems` information as well, just like the QML sample has.